documents/scripts/SQLScript_tinytitletinder.sql

```diff
@@ -250,4 +250,30 @@ END //
 DELIMITER ;
 
 
-call GetNamesWithNoParentRelation(1)
\ No newline at end of file
+#------------------ EVENTS ------------------
+
+DROP EVENT IF EXISTS changeUserToInactive;
+-- Change user_active to 0 after last login being over 1 year old
+DELIMITER //
+CREATE EVENT changeUserToInactive
+    ON SCHEDULE
+        EVERY 1 DAY STARTS CURRENT_TIMESTAMP
+    DO BEGIN
+    UPDATE user
+    SET user_active = 0
+    WHERE last_login < NOW() - INTERVAL 1 YEAR
+      AND user_active = 1;
```

Comment:
Since user_active has changed to boolean:
```suggestion
    SET user_active = false
    WHERE last_login < NOW() - INTERVAL 1 YEAR
      AND user_active = true;
```